.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
Are you intending to check in the `.vscode` folder?  Just wanting to make sure this was intentional, no big objection.

Reply:
Unintentional. I'll add to gitignore